controllers/rollingupgrade_controller.go

```diff
@@ -846,34 +846,19 @@ func (r *RollingUpgradeReconciler) validateRollingUpgradeObj(ruObj *upgrademgrv1
 
 // setDefaultsForRollingUpdateStrategy sets the default values for type, maxUnavailable and drainTimeout
 func (r *RollingUpgradeReconciler) setDefaultsForRollingUpdateStrategy(ruObj *upgrademgrv1alpha1.RollingUpgrade) {
-
-	// Setting the default values for the update strategy when strategy is not set
-	// Default behaviour should be to update one node at a time and should wait for kubectl drain completion
-	var nilStrategy = upgrademgrv1alpha1.UpdateStrategy{}
-	if ruObj.Spec.Strategy == nilStrategy {
```

Comment:
no need to do if/else because new object is always nilStrategy, so simplify code path.